tests/testfiles/CommonMark_0.30/autolinks/002.test

```diff
@@ -10,7 +10,7 @@
 documentBegin
 BEGIN link
 - label: http://foo.bar.baz/test?q=hello(ampersand)id=22(ampersand)boolean
-- URI: http://foo.bar.baz/test?q=hello(ampersand)id=22(ampersand)boolean
+- URI: http://foo.bar.baz/test?q=hello&id=22&boolean
```

Comment:
Since URLs can be used as either programming strings or typographic material, we produce both escaped and non-escaped variants, see <https://witiko.github.io/markdown/#link-renderer>. The non-escaped variant only escapes the left brace (`{`), the right brace (`}`), and the backslash (`\`).

However, in unit tests, we only show the non-escaped variant for improved readability, see [file `tests/support/plain-setup.tex`][1].

 [1]: https://github.com/Witiko/markdown/blob/c9ae738aeef79668f28177125e0c7c9c1517601e/tests/support/plain-setup.tex#L52-L57